setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.2.0
+current_version = 0.4.0
```

Comment:
Aha, so actually the new version number comes from a bumpversion script? If we do it like this, we don't need the single-sourcing in `setup.py`, but we definitely need dev docs telling everyone how to exactly do that. I wouldn't know atm. What are the advantages of this approach over updating the version in `__init__.py` manually?

Reply:
Yeah, I don't know either. that was in the cookiecutter but I'd just rather go with yours and remove the bumpversion dependency.